libs/core/src/lib/nested-list/prepared-nested-list/prepared-nested-list.component.ts

```diff
@@ -40,28 +47,28 @@ export class PreparedNestedListComponent implements AfterViewInit {
     @ViewChild(forwardRef(() => NestedListDirective))
     _nestedListDirective: NestedListDirective;
 
-    /**
-     * @hidden
-     */
-    @ViewChildren(forwardRef(() => NestedItemDirective))
-    nestedListItems: QueryList<NestedItemDirective>;
-
     /**
      * In prepared nested list, nested items should be taken as reference of View, not Content.
      * There is direct reference to these directives here.
      */
     get nestedListDirective(): NestedListDirective {
-        return Object.assign(this._nestedListDirective, { nestedItems: this.nestedListItems });
+        return this._nestedListDirective;
     }
 
     /** @hidden */
     constructor (
-        private changeDetRef: ChangeDetectorRef
+        private _changeDetRef: ChangeDetectorRef,
+        @Optional() private _nestedItemService: NestedItemService
     ) {}
 
     /** @hidden */
     ngAfterViewInit(): void {
-        this.changeDetRef.markForCheck();
-        this.changeDetRef.detectChanges();
+        this._changeDetRef.markForCheck();
+        this._changeDetRef.detectChanges();
```

Comment:
Does it make sense to call `this._changeDetRef.markForCheck();` right before forcing change detection?